docu/docs/config.md

```diff
@@ -157,15 +157,16 @@ decoder:
 ```
 
 ---
-## Server
+<h2 id="server-configuration">Server</h2>
```

Comment:
Wieso als H2? Geht das nicht auch mit `file.md#headline`?

Reply:
@Schrolli91 #server linked auf die erste gefundene Instanz von `server` in einer Headline, das wäre in diesem Fall falsch. Alternativ koennte ich die Headline auf `Server-Konfiguration` o.ä. ändern, dann koennten wir unten eindeutig darauf verweisen :)

Reply:
Finde halt das HTML etwas störend im sonst recht sauberen Markdown.
Aber wenn das nicht anders geht, ohne jetzt alles irgendwie umzubenennen, dann lassen wir das halt so.
Außer du hast noch eine Idee - bin mir nicht sicher ob mkDocs da nicht auch eine Möglichkeit bietet, benutzerdefinierte Anker zu erzeugen.

Reply:
@Schrolli91 Ich wüsste leider so keine bessere Lösung, ausser die Section umzubenennen. Wird auch häufig so (oder mit <a id="section"></a> hinter der heading) verwendet. Was meinst du?